### PR TITLE
[#463] Create composer version of WaterfallHostBotDotNet

### PR DESCRIPTION
**Pull request by NahuelPerales**
Addresses # 463

## Description
This PR adds the WaterfallHostBot Composer version  and updates the tests to cover it.

### Detailed Changes
- Create **WaterfallHostBotComposer** project.
- Updates test projects to cover the new bot.
- Updates bots dependencies to 4.15.0 version.

## Testing
These images show some of the processes working as expected on Emulator.

image

image (33)

image

image

### CODE REVIEW

File: Bots/DotNet/Skills/CodeFirst/EchoSkillBot-v3/Web.config [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (JuanAr) [inline Bots/DotNet/Skills/CodeFirst/EchoSkillBot-v3/Web.config:1]: Why was this file modified?

File: Bots/DotNet/FunctionalTestsBots.sln [hunk 2]
```diff
@@ -35,8 +35,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleHostBotComposer", "Co
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EchoSkillBotComposer", "Skills\Composer\EchoSkillBotComposer\EchoSkillBotComposer.csproj", "{9BA47CF9-7D90-4B5C-A9FA-01797A435D53}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ComposerHostBotDotNet", "Consumers\Composer\ComposerHostBotDotNet\ComposerHostBotDotNet.csproj", "{E3FCDDD4-553D-44BA-8B2F-82BA778261BC}"
+EndProject
```
Comment (JuanAr) [inline Bots/DotNet/FunctionalTestsBots.sln:39]: Move adding this project to the solution to PR101

File: Bots/DotNet/FunctionalTestsBots.sln [hunk 3]
```diff
@@ -35,8 +35,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleHostBotComposer", "Co
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EchoSkillBotComposer", "Skills\Composer\EchoSkillBotComposer\EchoSkillBotComposer.csproj", "{9BA47CF9-7D90-4B5C-A9FA-01797A435D53}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ComposerHostBotDotNet", "Consumers\Composer\ComposerHostBotDotNet\ComposerHostBotDotNet.csproj", "{E3FCDDD4-553D-44BA-8B2F-82BA778261BC}"
+EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ComposerSkillBotDotNet", "Skills\Composer\ComposerSkillBotDotNet\ComposerSkillBotDotNet\ComposerSkillBotDotNet.csproj", "{925E0E3F-194D-4E56-AE78-6FF6D49B7E7E}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WaterfallHostBotComposer", "Consumers\Composer\WaterfallHostBotComposer\WaterfallHostBotComposer.csproj", "{80AF5608-8126-439F-B463-197E6A17D941}"
+EndProject
```
Comment (JuanAr) [inline Bots/DotNet/FunctionalTestsBots.sln:43]: Was the project already in the solution? Is not a new project right?

**Review by JuanAr — CHANGES_REQUESTED** [review]
Just move adding ComposerHostBotDotNet to the solution to PR101

**Comment by NahuelPerales** [comment]
This PR was restructured in [PR 106](https://github.com/southworks/BotFramework-FunctionalTests/pull/106)

**Comment by NahuelPerales** [comment]
This PR was restructured in [PR 106](https://github.com/southworks/BotFramework-FunctionalTests/pull/106)

**Comment by NahuelPerales** [comment]
This PR was restructured in [PR 105](https://github.com/southworks/BotFramework-FunctionalTests/pull/105) and [PR 107](https://github.com/southworks/BotFramework-FunctionalTests/pull/107).
